.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: build-docs
+on:
+  pull_request:
+    paths:
+      - .github/workflows/**
+      - docs/**
+  push:
+concurrency:
+  # Cancel other jobs in the same branch. We don't care whether CI passes
+  # on old commits.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+jobs:
+  build-docs:
+    if: github.event_name == 'pull_request'
```

Comment:
```suggestion
```

Reply:
I've copied this straight from ci.yml. Are you sure that's redundant?

Reply:
checked other yamls and this really seems redundant